.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -113,9 +113,9 @@ body:
       label: What operating system is DMS running on?
       options:
         - Linux
-        - macOS
-        - Windows
-        - Other
+        - macOS (not officially supported)
+        - Windows (unsupported)
+        - Other (unsupported)
```

Comment:
Is there a difference between "not officially supported" and "unsupported"?

Reply:
The idea was that some maintainers will be able to provide support here, while for `unsupported`, there are no maintainers to my knowledge that support these cases. I will add an explanation at the top.